source/Engine.cpp

```diff
@@ -942,7 +942,13 @@ void Engine::Draw() const
 		if(messagePoint.Y() < messageBox.Top())
 			break;
 		float alpha = (it->step + 1000 - step) * .001f;
-		Color color(alpha, 0.f);
+		Color color;
+		if (it->importance == Messages::Importance::Highest) {
+			color = Color::Red.Additive(alpha);
+		}
+		else {
+			color = Color(alpha);
+		}
```

Comment:
The color used shouldn't be hardcoded, and should instead be defined in the interfaces file as to allow it to be easily customized. (And by extension, we shouldn't be hardcoding any colors into the Color class as you're doing.)
https://github.com/endless-sky/endless-sky/blob/d712e94e3923a81dd1ab14afcc2339ee86790740/data/interfaces.txt#L92-L95

Reply:
Ok makes sense, thanks; will update.

Reply:
Done